src/test/java/duke/TaskListTest.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class TaskListTest {
+    @Test
+    public void findTaskUsingKeyword_test1() throws FileNotFoundException {
```

Comment:
Good name for your test method! I think you could use the same idea to name your other tests.